src/cargo/util_schemas/mod.rs

```diff
@@ -0,0 +1,3 @@
+//! Low-level Cargo format schemas
```

Comment:
So there will be more types in `util_schemas`. I guess types for `.carg/config.toml` and `Cargo.toml` will all end up there? Are we going to have feature flags for different types?

It's also not clear what low-level actually means here, from a contributor's perspective.

Reply:
Added a bit more of an explanation.

My assumption is these will be light enough to not need feature flags.